lab1.js

```diff
@@ -117,18 +132,24 @@ var tooHungryDay;
  pondering protein supplements (the first day the average dips below 4
  meals)
 */
+tooHungryDay = 0;
 
-// your code goes here
+for (var i = 0; i < mealsPerDay.length; i++) {
+  console.log(mealsPerDay[i]);
+  if (mealsPerDay[i] < 4) {
+    tooHungryDay++;
```

Comment:
here, you don't really need to increment. Remember, we're just trying to find the first day that the average falls below 4. So you can just assign i (or i+1, since us humans usually count starting at 1, not 0) to tooHungryDay and then break out of the loop.